packages/mui-base/src/useButton/useButton.types.ts

```diff
@@ -12,18 +12,16 @@ export interface UseButtonParameters {
    * @default false
    */
   focusableWhenDisabled?: boolean;
-  href?: string;
   onFocusVisible?: React.FocusEventHandler;
   rootRef?: React.Ref<Element>;
   tabIndex?: NonNullable<React.HTMLAttributes<any>['tabIndex']>;
-  to?: string;
   /**
    * Type attribute applied when the `component` is `button`.
    * @default 'button'
    */
   type?: React.ButtonHTMLAttributes<HTMLButtonElement>['type'];
   /**
-   * The HTML element, e.g.'button', 'a' etc
+   * The HTML element, e.g.'button', 'input' etc
```

Comment:
```suggestion
   * The HTML element, for example, 'button', 'input', etc.
```

Reply:
So it's not misleading, maybe we can add here examples like, 'Link' component instead of button or input. What do you think?

Reply:
I thought we were doing this split so that developers wouldn't use router links in our Button anymore. If there's a need to use an external (like Next.js) Link component, they should use it directly.